crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -42,8 +49,8 @@ pub struct SequencerBlockData {
     pub(crate) header: Header,
     /// This field should be set for every block with height > 1.
     pub(crate) last_commit: Option<Commit>,
-    /// namespace -> rollup txs
-    pub(crate) rollup_txs: HashMap<Namespace, Vec<Vec<u8>>>,
+    /// namespace -> rollup data (chain ID and transactions)
+    pub(crate) rollup_data: HashMap<Namespace, RollupData>,
```

Comment:
debating just changing this into `HashMap<Vec<u8>, Vec<Vec<u8>>>` where the key is the `chain_id` and the value is the transactions, and then only doing chain_id -> namespace when we're constructing the `RollupNamespaceData`

Reply:
I'm leaning towards separating it out b/c this type should be generic/usable for any DA layer while `Namespace` here is celestia-specific, it also makes it easier/cleaner to implement some of the verification logic. I'll open a separate PR for it and see how you feel

Reply:
actually it doesn't really matter, I can leave as-is. I think we need to open up a bigger discussion about what we're gossiping around and the format of it, as that also matters for verification.